src/scene.rs

```diff
@@ -166,6 +188,15 @@ impl<I: ImageSize> Scene<I> {
         total
     }
 
+    /// Get the number of running animations for a sprite. If sprite does not
+    /// exist, returns 0.
```

Comment:
I feel like this method should return an Option, and not a sentinel value of 0 for a failure state.